script.js

```diff
@@ -42,26 +39,26 @@ element2.ondragstart = (ev) => {
 // Add color to fields  to highlight them
 // where mouse is and fields on right to mouse
 function highlightFields(ev) {
-  const targetEl = ev.target
-  let el = targetEl
+  ev.preventDefault()
 
-  if (checkWrap(targetEl) || checkPlaced(targetEl)) {
+  if (checkWrap(ev.target) || checkPlaced(ev.target)) {
```

Comment:
when you mix functionality commits and refactoring commits in 1 PR, your reviewer will review the same thing more than once which makes a lot of context switching 🙈